sssd_test_framework/utils/sss_override.py

```diff
@@ -0,0 +1,402 @@
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "SSSOverrideUtils",
+    "SSSOverrideUser",
+    "SSSOverrideGroup",
+]
+
+
+class SSSOverrideUtils(MultihostUtility[MultihostHost]):
+    """
+    Management of local override users and groups, using sss_override.
+    """
+
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
+        """
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        :param fs: Linux file system
+        :type fs: LinuxFileSystem
+        """
+        super().__init__(host)
+
+        self.cli: CLIBuilder = CLIBuilder(host.ssh)
+        self.fs: LinuxFileSystem = fs
+
+    def user(self, name: str) -> SSSOverrideUser:
+        """
+        Get local override user object.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.Client)
+            def test_example(client: Client):
```

Comment:
I'm sorry, this example still does not work. Please, try to actually run it. The client topology will not work because you need to create a user. So the topology should be for example LDAP and you need to create the user that you override.

```python
@pytest.mark.topology(KnownTopology.LDAP)
def test_override(client: Client, ldap: LDAP):
    # Add user
    ldap.user("user-1").add(uid=10001, gid=10001, gecos="gecos")

    # SSSD must be running for sss_override to work
    client.sssd.start()

    # Create local override for the user
    client.sss_override.user("user-1").add(name="o-user-1", uid=20001, gid=20001, gecos="o-gecos")

    # SSSD must be restarted so newly created view can be applied
    client.sssd.restart()

    # Check the result
    result = client.tools.getent.passwd("o-user-1")
    assert result is not None
    assert result.uid == 20001
    assert result.gid == 20001
    assert result.gecos == "o-gecos"
```

Reply:
Same for group.